src/app/giftcards/api.service.js

```diff
@@ -0,0 +1,28 @@
+angular.module('giftCardsModule')
+
+    .service('giftcardsApiService', [
+        "$resource",
+        'REST_SERVER_URI',
+        function($resource, REST_SERVER_URI) {
+
+            return $resource(REST_SERVER_URI, {}, {
+                // How much money is on this giftcard
+                "getBalance": {
+                    method: "GET",
+                    url: REST_SERVER_URI + "/giftcards/:code"
+                },
+                // Attach the gift card to this session
+                "apply": {
+                    method: "GET",
+                    url: REST_SERVER_URI + "/giftcards/:code/apply"
+                },
+                // Remove giftcard from checkout
+                "neglect": {
+                    method: "GET",
+                    url: REST_SERVER_URI + "/giftcards/:code/neglect"
```

Comment:
I know this is here - I will put in a PR to change the endpoint name to **remove** or **delete**